src/mantle/support/class-html.php

```diff
@@ -152,6 +152,36 @@ public function first_by_selector( string $selector ): static {
 		return $this->filter( $selector )->first();
 	}
 
+	/**
+	 * Get the nearest previous sibling element.
+	 *
+	 * @param string|null $selector Optional CSS selector to filter the previous siblings.
+	 * @throws \InvalidArgumentException If the current node list is empty.
+	 */
+	public function previousSibling( ?string $selector = null ): static {
```

Comment:
These should be underscore_cased -- this class does have a mix of them because of the extended class but I was trying to make our methods use underscore_case.